src/app/pages/StakingPage/Features/ValidatorList/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -93,49 +190,105 @@ exports[`<ValidatorList  /> empty should match snapshot 1`] = `
 }
 
 @media only screen and (max-width:768px) {
-  .c1 {
+
+}
+
+@media only screen and (max-width:768px) {
+  .c4 {
     padding: 12px;
   }
 }
 
 @media only screen and (max-width:768px) {
-  .c2 {
+  .c1 {
+    margin-bottom: 6px;
+  }
+}
+
+@media only screen and (max-width:768px) {
+  .c5 {
     margin-top: 0px;
   }
 }
 
 @media only screen and (max-width:768px) {
-  .c2 {
+  .c5 {
     font-size: 26px;
     line-height: 32px;
     max-width: 624px;
   }
 }
 
+@media only screen and (max-width:768px) {
+  .c2 {
+    padding: 6px;
+  }
+}
+
 <div
   class="c0"
   style="min-height: 100dvh;"
 >
-  <div
+  <nav
     class="c1"
   >
-    <h1
+    <a
       class="c2"
+      href="/account/undefined/stake"
+    >
+      <button
+        class="c3"
+        tabindex="-1"
+        type="button"
+      >
+        account.subnavigation.validators
+      </button>
+    </a>
+    <a
+      class="c2"
+      href="/account/undefined/active-delegations"
```

Comment:
Can we add data-testid to elements that were root previously in all views that now use StakeSubnav?
In tests we could switch from `(component.baseElement).toMatchSnapshot()` to `expect(screen.getByTestId('WHATEVER')).toMatchSnapshot()`. I just don't see a value in extra 1.3l lines of snapshots here.

Reply:
+1